docs/360.html

```diff
@@ -29,11 +29,7 @@
 
       player = cld.videoPlayer('player', {
         publicId: 'video-player/Golden-Gate-Bridge',
-        sourceTypes: ['mp4'],
-        transformation: {
```

Comment:
Looks like a bug. A transformation was provided but it's not as part of the URL.
it has been fixed. 
![Screen Shot 2022-02-17 at 17 02 24](https://user-images.githubusercontent.com/76470570/154509058-a0fd206d-c766-4a77-90a3-3f60074f252b.png)